shim/linux/src/entry.c

```diff
@@ -66,6 +87,34 @@ dispatch_system_kvm_check_extension(void)
 static long
 dispatch_system_kvm_create_vm(void)
 {
+    char vmname[22];
+    int32_t fd;
+    struct shim_vm_t *vm =
+        (struct shim_vm_t *)vmalloc(sizeof(struct shim_vm_t));
+    if (NULL == vm) {
+        bferror("dispatch_system_kvm_create_vm : vm vmalloc failed");
+        goto vm_free;
+    }
+    if (handle_system_kvm_create_vm(vm)) {
+        bferror(
+            "dispatch_system_kvm_create_vm : handle_system_kvm_create_vm "
```

Comment:
remove "dispatch_system_kvm_create_vm " from all of the bferror calls. This should just be:
```
bferror("handle_system_kvm_create_vm failed");
```

The bferror macro will add the remaining information for you automatically.

Reply:
Updated